DivanteLtd/vue-storefront
# remove prefech dayjs locales

### Related issues


closes #3498

### Short description and why it's useful

Added function that creates regexp with needed `dayjs` locales

### QA
1. open http://next.storefrontcloud.io, clear cache and refresh page
2. open Elements tab in devtools and expand `head` tag. There should be loaded a lot of `dayjs...js` files
3. checkout on current branch, prepare env (`yarn build` and `yarn start`)
4. make 1 and 2 step on current branch page
5. In `head` should be loaded only needed `dayjs...js` files based on config. 
6. Previous steps could be tested with different config: 
 - without specified defaultLocale in `config.i18n.defaultLocale` (should load only `en` file, because it's fallback language)
 - without `multistore` config (should load `en` and one specified in `config.i18n.defaultLocale`)
 - with `multistore` config (should load `en` and all specified in storeView `i18n.defaultLocale`)

### Which environment this relates to
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)


### Upgrade Notes and Changelog


**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and currently important rules acceptance
